components/ReplyFeedback.js

```diff
@@ -291,28 +280,22 @@ function ReplyFeedback({
 
   return (
     <div className={cx(classes.root, className)}>
-      <button
-        className={cx(classes.vote, ownVote === 'UPVOTE' && classes.voted)}
-        type="button"
-        onClick={e => openVotePopover(e, 'UPVOTE')}
-        data-ga="Upvote"
-      >
-        <ThumbUpIcon className={classes.outlinedIcon} />
-      </button>
-      <button
-        className={cx(classes.vote, ownVote === 'DOWNVOTE' && classes.voted)}
-        type="button"
-        onClick={e => openVotePopover(e, 'DOWNVOTE')}
-        data-ga="Downvote"
-      >
-        <ThumbDownIcon className={classes.outlinedIcon} />
-      </button>
       <div className={classes.buttonGroup} data-ga="Number display">
-        <button className={classes.vote} type="button">
+        <button
```

Comment:
The height of the buttons are not even. 
![image](https://user-images.githubusercontent.com/108608/85226443-0ed90e00-b40a-11ea-94ff-9ac274f2f5bf.png)

When hovering on the button, the border highlight is scattered:
![image](https://user-images.githubusercontent.com/108608/85226612-f9181880-b40a-11ea-999d-b5cafabf52fb.png)

The same goes to the button groups in `CreateReplyRequestForm`. They have different heights (36px vs 38px).
![imba-buttons](https://user-images.githubusercontent.com/108608/85226538-9aeb3580-b40a-11ea-96d9-9167d5db21fb.gif)

I would suggest we use [outlined button groups](https://material-ui.com/components/button-group/#sizes-and-colors) for all the connected buttons buttons, and [`<Button variant="contained" color="primary" disableElevation>`](https://material-ui.com/components/buttons/#contained-buttons) for "Reply to this message" button.

In this way we get even-height buttons, more sensible border highlight when hovering on the button, and a consistent feedback when clicking on the buttons.